kickstarts/base.ks.erb

```diff
@@ -1,5 +1,8 @@
 # Kickstart file automatically generated by anaconda.
-# Run the text based installer to see dependency info in screenshots
+
+# By default run the text based installer to see dependency info in screenshots
+<% if ENV['KS_GUI'] == '1' then "" else "text" end %>
```

Comment:
Grr, I did test this, but apparently not properly. I'll update this PR and submit a new one for master.